src/lib/Preview.js

```diff
@@ -1269,6 +1314,100 @@ class Preview extends EventEmitter {
         this.viewer.load(err);
     }
 
+    /**
+     * Create a generic log Object.
+     *
+     * @private
+     * @return {Object} Log details for viewer session and current file.
+     */
+    createLog() {
```

Comment:
to follow Tony's comment below, maybe this becomes `createEvent`?

Reply:
`createLogEvent()`? Since it contains relevant logging values?